src/repr/src/row.rs

```diff
@@ -580,7 +580,13 @@ impl Row {
     }
 
     // TODO(justin): find a better place to put this.
```

Comment:
nit: probably delete the todo, if you like? This seems like a very reasonable place to put this, now that it is properly marked as `unsafe`.

Reply:
I'm happy to do that, but didn't want to get in front of @justinj if there was a specific plan.